portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
-#content {
+/*#content {
```

Comment:
Delete the commented out code if you don't need it.

Reply:
Understood i havent deleted it yet because im taking a break from the css to focous on just getting content onto the rest of the page

Reply:
Fixing up the CSS in a separate PR sounds good, but you'll need to remove the commented out code before I can approve this PR. Checking in dead code isn't allowed.